src/Symfony/Component/Process/PhpExecutableFinder.php

```diff
@@ -37,7 +37,7 @@ public function find($includeArgs = true)
     {
         // HHVM support
         if (defined('HHVM_VERSION')) {
-            return (false !== ($hhvm = getenv('PHP_BINARY')) ? $hhvm : PHP_BINARY).($includeArgs ? ' '.implode(' ', $this->findArguments()) : '');
+            return (false !== (getenv('PHP_BINARY')) ?: PHP_BINARY).($includeArgs ? ' '.implode(' ', $this->findArguments()) : '');
```

Comment:
Fixed in https://github.com/symfony/symfony/commit/6e1ae53feaafd0bb5455b0737ac45106f0fd3c26